src/features/home/components/Filter/Filter.js

```diff
@@ -0,0 +1,236 @@
+      }
+    }
+  }, [currentPath, currentSort, history, top]);
+
+  return (
+    <div className={classes.buttonsOuterContainer}>
+      <Grid container className={clsx(classes.buttonContainer, className)}>
+        <Select
+          className={classes.select}
+          id="pot-type-select"
+          onChange={handlePathChange}
+          value={currentPath}
+          disableUnderline
+          IconComponent={iconComponent}
+          MenuProps={{
+            classes: {
+              paper: classes.menuStyle,
+              root: classes.selectRoot,
+            },
+            anchorOrigin: {
+              vertical: 'bottom',
+              horizontal: 'left',
+            },
+            transformOrigin: {
+              vertical: 'top',
+              horizontal: 'left',
+            },
+            getContentAnchorEl: null,
+          }}
+        >
+          {PotTypes.map(type => (
+            <MenuItem key={type.key} value={type.path}>
+              <div style={{ display: 'flex' }}>
+                <Typography className={classes.selectLabel}>{t('potType')}&nbsp;</Typography>
+                <Typography
+                  className={
+                    selected === type.key ? classes.selectValueSelected : classes.selectValue
+                  }
+                >
+                  {t(type.label)}
+                </Typography>
+              </div>
+            </MenuItem>
+          ))}
+        </Select>
+        <Select
+          className={classes.select}
+          id="pot-type-select"
+          onChange={handleSortChange}
+          value={currentSort}
+          disableUnderline
+          IconComponent={iconComponent}
+          MenuProps={{
+            classes: { paper: classes.menuStyle },
+            anchorOrigin: {
+              vertical: 'bottom',
+              horizontal: 'left',
+            },
+            transformOrigin: {
+              vertical: 'top',
+              horizontal: 'left',
+            },
+            getContentAnchorEl: null,
+          }}
+        >
+          {top !== 'my-moonpots'
+            ? sortByTypes.map(type => (
+                <MenuItem key={type.key} value={type.path}>
+                  <div style={{ display: 'flex' }}>
+                    <Typography className={classes.selectLabel}>{t('sortBy')}&nbsp;</Typography>
+                    <Typography
+                      className={
+                        sort === type.key ? classes.selectValueSelected : classes.selectValue
+                      }
+                    >
+                      {t(type.label)}
+                    </Typography>
+                  </div>
+                </MenuItem>
+              ))
+            : mySortByTypes.map(type => (
+                <MenuItem key={type.key} value={type.path}>
+                  <div style={{ display: 'flex' }}>
+                    <Typography className={classes.selectLabel}>{t('sortBy')}&nbsp;</Typography>
```

Comment:
Labe; should be 'Pot status' not 'Sort by' when 'My Pots' is selected

Reply:
Fixed this in the last commit